test/SlowTests/Issues/RavenDB_22013.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.ServerWide;
+using Raven.Server.ServerWide.Context;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_22013 : RavenTestBase
+    {
+        public RavenDB_22013(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [RavenFact(RavenTestCategory.None)]
+        public async Task ValidateNodesNotNullInDatabaseRecordBeforeSavingRecordClusterWide()
+        {
+            var databaseName = GetDatabaseName();
+            using (Server.ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            {
+                context.OpenReadTransaction();
```

Comment:
The context is probably isn't needed here?

Reply:
True :) fixed